packages/shared/src/modules/routes/libs/validation-schemas/libs/constants/routes-validation-config.constant.ts

```diff
@@ -0,0 +1,6 @@
+const ROUTES_VALIDATION_CONFIG = {
```

Comment:
Even though I agree it should be in constants by logic, we are currently using enum for validation rules, so let's follow this convention and  move it to enums instead of libs, and it should be in routes/libs/enums/route-validation-rule.enum.ts alongside the route-validation-message.enum.ts, not under validation-schema folder
```suggestion
const RouteValidationRule = {
```